src/js/providers/html5.js

```diff
@@ -125,6 +128,12 @@ function VideoProvider(_playerId, _playerConfig) {
                 offset: offset
             });
             _setPositionBeforeSeek(offset);
+            _checkForBuffering(offset);
+        },
+
+        seeked() {
+            VideoEvents.seeked.call(_this);
+            _videotag.removeEventListener('waiting', setBufferingState);
```

Comment:
Remove 'waiting' before `VideoEvents.seeked`. `VideoEvents.seeked` triggers `MEDIA_SEEKED` which could result in additional code running in an event handler. We should always remove event listeners first, and run code that dispatches events as late as possible.